test/test_service.py

```diff
@@ -825,7 +825,7 @@ def on_received_response(response):
 
     @with_main_loop
     def test_list_applications_contains_video_app(self, quit_cb):
-        '''/v1/list_applications should contain video app.'''
+        '''/v2/list_applications should contain video app.'''
```

Comment:
In this commit you're just refactoring the routes so that a version is passed, but you keep the version as `v1`. Yet in the tests all the versions are updated to `v2`. I don't think that's going to work.

Reply:
You're right - I should move all these tests to the other commit that introduced `v2`. Good catch.